wire-ios/Wire-iOS/Sources/UserInterface/Settings/DeviceManagement/ClientTableViewCell.swift

```diff
@@ -75,19 +56,13 @@ class ClientTableViewCell: UITableViewCell, DynamicTypeCapable {
             } else if userClient.isLegalHoldDevice {
                 nameLabel.text = L10n.Localizable.Device.Class.legalhold
             }
-
+            let proteusId = userClient.displayIdentifier.fingerprintStringWithSpaces.uppercased()
+            proteusIdLabel.text =  deviceDetailsSection.Proteus.id + ": " + proteusId
+            let mlsThumbPrint = userClient.mlsPublicKeys.ed25519?.fingerprintStringWithSpaces.uppercased() ?? ""
+            mlsThumbprintLabel.text =  deviceDetailsSection.Mls.title + ": " + mlsThumbPrint
+            proteusVerficiationStatusImageView.image = userClient.verified ? verifiedImage : .none
```

Comment:
question: Is there any other way to construct the strings other than doing it in code? I would like us to move away from such paradigm because it can lead to problems down the line. If there is no other way we can keep as is for now.